### PR TITLE
fix repeatLastTest

### LINKED ISSUE #21: repeatLastTest documentation is wrong
The README specifies that to run the last test you send the action: `{repeatLastTest: true}`. 
Doing so gives the error:
```
Error: no matching action found for {"repeatLastTest":true}
```

The feature tests also reference `operation` which is supposedly outdated

### PR DESCRIPTION
**Pull request by charlierudolph**
@kevgo 

resolves #21

### CODE REVIEW

File: src/command-runner.ls [hunk 1]
```diff
@@ -32,7 +32,7 @@ class CommandRunner
       @re-run-last-test command if @current-command
       return
 
-    if command.operation is 'repeatLastTest'
+    if command.repeatLastTest
```
Comment (kevgo) [inline src/command-runner.ls]: if command.repeat-last-test

**Comment by kevgo** [comment]
Thanks @charlierudolph! Sorry for not updating the documentation. Time to verify it via TutorialRunner!

**Comment by kevgo** [comment]
🚢

**Comment by kevgo** [comment]
Please ignore the failed Travis build, I'm just experimenting with setting up a build matrix against different Node versions. Main development will remain on Circle.

File: src/command-runner.ls (hunk 1 above)
Comment (charlierudolph) [inline src/command-runner.ls]: updated

**Comment by charlierudolph** [comment]
@kevgo I updated the release notes. Please do a quick review of the message.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Tertestrial Release Notes
 
+## Master
+
+- update command which triggers a re-run of the last test to match documentation
```
Comment (kevgo) [inline RELEASE_NOTES.md]: I would say it a bit more succinctly:

> - update "repeatLastTest" command to new format

**Comment by kevgo** [comment]
Thanks for doing this! 🚢

File: RELEASE_NOTES.md (hunk 2 above)
Comment (charlierudolph) [inline RELEASE_NOTES.md]: updated

**Pull request merged**